Sources/SwiftDiscord/Audit/DiscordAuditLogEntry.swift

```diff
@@ -113,7 +113,7 @@ public enum DiscordAuditLogActionType : Int {
     case memberPrune = 21
 
     /// Member ban add.
-    case memberBanAdd =	22
```

Comment:
Thanks, this was some weird copy-paste that was going on with AppCode, I guess I missed this one.